js/main.js

```diff
@@ -1,16 +1,34 @@
+// variable definiitions
 var $photoUrlInput = document.querySelector('#photoUrl');
 var $photo = document.querySelector('.photo');
 var $textInputs = document.querySelectorAll('.text-adjust');
+var $noEntriesText = document.querySelector('.no-entries-text');
+var $formElement = document.querySelector('form');
+var journalFeedList = document.querySelector('ul');
+var $navEntries = document.querySelector('.nav-entries');
+var $navNewEntry = document.querySelector('.new-button');
+var $newEntryForm = document.querySelector('.entry-form');
+var $entriesPage = document.querySelector('.entries');
 
+// Adding event listeners
 $photoUrlInput.addEventListener('input', changeUrl);
+$formElement.addEventListener('submit', formSubmitted);
+window.addEventListener('DOMContentLoaded', addingChildLoop);
+$navEntries.addEventListener('click', navToNewEntry);
+$navNewEntry.addEventListener('click', navToEntries);
 
+// Parsing local data
+var localData = {};
+var previousJournalJSON = localStorage.getItem('code-journal');
+if (previousJournalJSON) {
+  localData = JSON.parse(previousJournalJSON);
+}
```

Comment:
I'm not sure what the purpose of `localData` is, you should be using the `data` variable from `data.js`. You should also have this behavior of parsing from localStorage in `data.js` as well, so it is in the same file as the variable it is meant to be preserving and potentially reassigning the value of.